c/dataservice.c

```diff
@@ -164,8 +164,10 @@ static DataService *makeDataService(WebPlugin *plugin, JsonObject *serviceJsonOb
     service->initializer = NULL;
     printf("*** PANIC ***  service loading in METTLE not implemented\n");
 #else
-    printf("going for DLL EP lib=%s epName=%s\n",libraryName,initializerName);
-    service->initializer = (InitializerAPI*) lookupDLLEntryPoint(libraryName,initializerName);
+    char dllLocation[1023] = {0}; /* USS max path length */
+    snprintf(dllLocation, sizeof(dllLocation), "%s/%s", plugin->pluginLocation, libraryName);
```

Comment:
Right idea, but wrong path.
pluginLocation = root folder.
Server logic goes into root/lib
See https://github.com/zowe/zlux/wiki/ZLUX-App-filesystem-structure#lib for what we want to follow.


Additionally, plugin->pluginLocation is sadly not santized: https://github.com/zowe/zss/blob/master/c/zss.c#L443
It may or may not already have a trailing slash.

Reply:
Change has been made.